src/services/itemMembership/plugins/MembershipRequest/service.ts

```diff
@@ -1,11 +1,60 @@
 import { singleton } from 'tsyringe';
 
+import { Member, PermissionLevel } from '@graasp/sdk';
+
+import { BaseLogger } from '../../../../logger';
+import { MAIL } from '../../../../plugins/mailer/langs/constants';
 import { MailerService } from '../../../../plugins/mailer/service';
+import { BUILDER_HOST } from '../../../../utils/config';
+import { Repositories } from '../../../../utils/repositories';
+import { Item } from '../../../item/entities/Item';
 
 @singleton()
 export class MembershipRequestService {
   private readonly mailerService: MailerService;
-  constructor(mailerService: MailerService) {
+  private readonly log: BaseLogger;
+
+  constructor(mailerService: MailerService, log: BaseLogger) {
     this.mailerService = mailerService;
+    this.log = log;
+  }
+
+  async get({ membershipRequestRepository }: Repositories, memberId: string, itemId: string) {
```

Comment:
It is possible to pass only the needed repository here ?

```suggestion
  async get(membershipRequestRepository: MembershipRequestRepository, memberId: string, itemId: string) {
```